exolve.html

```diff
@@ -396,6 +396,8 @@
              <b>Ctrl-B:</b> Print crossword, <b>Ctrl-/:</b> Jump to/back-from
              notes, <b>Ctrl-*:</b> Mark clue as fave in notes, adding a *
              prefix.</li>
+         <li><b>Delete:</b>
+						 Clear the content sof the current square.</li>             
```

Comment:
just re-copy the updated exolve-m.js into this file after fixing the minor typos listed above.

Reply:
updated